models/emergency_contact.py

```diff
@@ -9,28 +8,18 @@ class EmergencyContactModel(BaseModel):
     __tablename__ = "emergency_contacts"
 
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(100))
+    name = db.Column(db.String(100), nullable=False, unique=True, index=True)
     description = db.Column(db.String(256))
-    #relationships
-    contact_numbers = relationship('ContactNumberModel', backref='contact_numbers', lazy=True)
 
-    def __init__(self, name, contact_numbers, description=None):
-        self.name = name
-        self.description = description if description else ''
-        self.contact_numbers = []
-        for number in contact_numbers:
-            item = ContactNumberModel(
-                emergency_contact_id = self.id,
-                number = number['number'],
-                numtype = number['numtype'] if 'numtype' in number.keys() else '',
-                extension = number['extension'] if 'extension' in number.keys() else '',
-            )
-            db.session.add(item)
-            self.contact_numbers.append(item)
+    contact_numbers = relationship(
+        'ContactNumberModel',
+        backref='contact_numbers',
```

Comment:
oops I think this is one of my commits, however this line should be:
```suggestion
        backref='emergency_contact',
```
That will allow us to ask a ContactNumber for its emergency_contact info